src/Raven.Server/Documents/ETL/EtlProcess.cs

```diff
@@ -229,16 +229,9 @@ public virtual IEnumerable<TExtracted> Extract(DocumentsOperationContext context
 
                             counters = new FilterCountersEnumerator(counters, stats, Database.DocumentsStorage, context);
 
-                            var counterTombstones = Database.DocumentsStorage.GetTombstonesFrom(context, CountersStorage.CountersTombstones, fromEtag, 0, int.MaxValue)
-                                .GetEnumerator();
-                            scope.EnsureDispose(counterTombstones);
-
-                            counterTombstones = new FilterTombstonesEnumerator(counterTombstones, stats, Tombstone.TombstoneType.Counter, context, maxEtag: lastDocEtag);
```

Comment:
We drop the concept of counter tombstones completely, right? What is the approach now?

Reply:
Do we ever remove those from the counter group?

Reply:
no we don't remove them, at least for now. we've discussed about trying to use the `TombstoneCleaner` to remove deleted counters that their 'change vector' is already merged. 
but this is not implemented yet